products/oracle-database.md

```diff
@@ -0,0 +1,124 @@
+
+-   releaseCycle: "19c"
+    lts: true
+    latest: "19.17.0.0.221018"
+    support: 2027-04-30
+    eol: false
+    latestReleaseDate: 2023-01-13
+    releaseDate: 2019-02-01
+    link: "https://docs.oracle.com/en/database/oracle/oracle-database/19/rnrdm/database-release-notes.pdf"
+
+-   releaseCycle: "18c"
+    lts: true
+    latest: "19.17.0.0.221018"
+    support: 2027-04-30
+    eol: false
+    latestReleaseDate: 2023-01-13
+    releaseDate: 2019-02-01
+    link: "https://support.oracle.com/knowledge/Oracle%20Cloud/18202104_9.html"
+  
+-   releaseCycle: "12c"
+    lts: false
+    latest: "12.2.0.1.210720"
+    support: 2017-03-01
+    eol: false
+    latestReleaseDate: 2022-04-01
+    releaseDate: 2013-07-01
+    link: "https://docs.oracle.com/en/database/oracle/oracle-database/12.2/ladbn/index.html"
+  
+-   releaseCycle: "11g"
+    lts: false
+    latest: "11.2.0.4"
+    support: 2015-01-01
+    eol: true
+    latestReleaseDate: 2022-09-13
+    releaseDate: 2007-09-01
+    link: "https://support.oracle.com/knowledge/Oracle%20Cloud/2068368_1.html"
+  
+-   releaseCycle: "10g"
+    lts: false
+    latest: "10.2.0.5"
+    support: 2010-07-01
+    eol: true
+    latestReleaseDate: 2005-07-01
+    releaseDate: 2004-01-01
+    link: https://docs.oracle.com/cd/B19306_01/nav/portal_1.htm
+  
+-   releaseCycle: "9i"
+    lts: false
+    latest: "9.2.0.8"
+    support: 2007-07-01
+    eol: true
+    latestReleaseDate: 2007-04-01
+    releaseDate: 2004-01-01
+    link: "https://www.oracle.com/database/technologies/oracle9i.html"
+
+-   releaseCycle: "8i"
+    lts: false
+    latest: "8.1.7.4"
+    support: 2004-12-01
+    eol: true
+    latestReleaseDate: 2008-08-01
+    releaseDate: 1997-06-01
+    link: "https://www.oracle.com/database/technologies/oracle8i.html"
+ 
+---
+
+## Product description
+
+> Oracle Database is a relational database management system (RDBMS) developed
+> by Oracle Corporation.
+
+> It is widely used in enterprise environments for managing large amounts of structured data,
+>such as financial information, customer data, and inventory management.
+
+> Oracle Database uses the Structured Query Language (SQL) for data manipulation and
+> management, and it provides advanced features such as robust security, transactional
+> consistency, and high availability. Additionally, Oracle Database offers a wide range
+> of tools and technologies for data warehousing, business intelligence, and application 
+> development, making it a versatile and powerful platform for businesses of all sizes.
+
+## Oracle Lifetime Support Policy
+
+> The "Lifetime Support Policy" provides access to technical experts for as long as
+> you license your Oracle products.
+> It consists of three support stages: Premier Support, Extended Support, and Sustaining Support.
+
+## Premier Support
+
+> Premier Support provides you with maintenance and support for your Oracle Database
```

Comment:
A more neutral and concise style should be preferred. You should avoid sentences with "you" or "your".

Example for this paragraph :

> Premier Support provides maintenance and support for five years from the general availability date of a given release.